moveit_ros/perception/occupancy_map_monitor/src/occupancy_map_monitor.cpp

```diff
@@ -280,7 +286,12 @@ bool OccupancyMapMonitor::getShapeTransformCache(std::size_t index, const std::s
         std::map<ShapeHandle, ShapeHandle>::const_iterator jt = mesh_handles_[index].find(it->first);
         if (jt == mesh_handles_[index].end())
         {
-          ROS_ERROR_THROTTLE(1, "Incorrect mapping of mesh handles");
```

Comment:
You can use RCUTILS_LOG_ERROR_THROTTLE https://github.com/ros2/rcutils/blob/master/test/test_logging_macros.cpp